hie.yaml

```diff
@@ -0,0 +1,225 @@
+cradle:
+  multi:
```

Comment:
Can we put `hie.yaml` files in the individual directories, or does it have to be at the root?

Reply:
Gotta be root IIRC. Otherwise it probably won't find the other packages. TBH, works like this, I don't wanna touch it.

Reply:
Okay. Can we at least sort by package and put a blank line between packages? It's a bit of a monster otherwise.